components/shooter.py

```diff
@@ -10,21 +10,21 @@ class Shooter:
     right_motor: ctre.TalonFX
 
     motor_speed = tunable(0.0)
+    COMPENSATED_VOLTAGE = 10.0
 
-    MAX_RP100ms = 10
-    pidF = 1023 / (2048 * MAX_RP100ms)
-    pidP = 0.15
+    pidF = 0.11468 / COMPENSATED_VOLTAGE * 1023 / 2048 * 10
```

Comment:
Let's make it clearer what's happening here :pray:

```suggestion
    pidF = 0.11468 / COMPENSATED_VOLTAGE * 1023 / RPS_TO_CTRE_UNITS
```